composer.json

```diff
@@ -73,7 +73,7 @@
         "friendsofsymfony/oauth-server-bundle": "^1.5",
         "stof/doctrine-extensions-bundle": "^1.2",
         "scheb/two-factor-bundle": "^4.4",
-        "grandt/phpepub": "dev-master",
+        "wallabag/phpepub": "^4.0.7.2",
```

Comment:
```suggestion
        "wallabag/phpepub": "^4.0",
```

Reply:
I don't agree with the version constraint change: there is a certain amount of changes since 4.0 (_and since the last 4.0.7.1_), maybe we should soon make a major bump.